### PR TITLE
Dan v2 updates

### PR DESCRIPTION
**Pull request by chillenberger**
- centralize all serverless pricing in one spot
- centralize all serverless models in one spot

### CODE REVIEW

File: pgml-dashboard/src/components/tables/serverless_models/mod.rs [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+#[template(path = "tables/serverless_models/template.html")]
+pub struct ServerlessModels {
+    style_type: String,
+    embedding_models: [Component; 4],
+    instruct_models: [Component; 5],
+    summarization_models: [Component; 1],
+}
+
+impl ServerlessModels {
+    pub fn new() -> ServerlessModels {
```
Comment (levkk) [inline pgml-dashboard/src/components/tables/serverless_models/mod.rs:16]: This feels like it belongs in our main app, not in the open source dashboard.

Comment (chillenberger) [reply pgml-dashboard/src/components/tables/serverless_models/mod.rs:16]: it is used in our documentation.

Comment (levkk) [reply pgml-dashboard/src/components/tables/serverless_models/mod.rs:16]: I understand. That requires us to fix our architecture to be able to inject data into the docs which may not be available in the open source repo.

Comment (chillenberger) [reply pgml-dashboard/src/components/tables/serverless_models/mod.rs:16]: https://github.com/postgresml/postgresml/issues/1492

**Pull request merged**